### ISSUE TITLE
[bug] Light theme colors contrast

### ISSUE THREAD

**Issue by gregmsanderson** [mehdibha/dotUI#13]
### Feature description

Hi,

I wonder if the contrast should be increased for some of the default colours to improve accessibility?

Perhaps white text on a darker red background?

![Screenshot 2024-07-11 at 14 45 40](https://github.com/mehdibha/dotUI/assets/10390169/785763d3-71cc-49db-83ba-f0285f5f5cb4)

![Screenshot 2024-07-11 at 14 46 26](https://github.com/mehdibha/dotUI/assets/10390169/5dc6956c-92bf-4ca3-a5d6-6e4f1d381643)

### Affected component/components

Button, Alert

### Additional Context

https://webaim.org/resources/contrastchecker/

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues and PRs

**Comment by mehdibha** [comment]
You're absolutely right.
I am currently working on a themes page, it will have pre-built themes and a theme generator with colors that are automatically contrasted using adobe leonardo under the hood. Dropping soon
